docs/reference/snapshot-restore/restore-snapshot.asciidoc

```diff
@@ -1,175 +1,268 @@
+
+# Reindex the restored index to rename it
+POST _reindex
+{
+  "source": {
+    "index": "restored-my-index"
+  },
+  "dest": {
+    "index": "my-index"
+  }
+}
+
+# Delete the original data stream
+DELETE _data_stream/logs-my_app-default
+
+# Reindex the restored data stream to rename it
+POST _reindex
+{
+  "source": {
+    "index": "restored-logs-my_app-default"
   },
-  "ignore_index_settings": [
-    "index.refresh_interval"
-  ]
+  "dest": {
+    "index": "logs-my_app-default",
+    "op_type": "create"
+  }
 }
------------------------------------
+----
 // TEST[continued]
 
-NOTE: Some settings such as `index.number_of_shards` cannot be changed during the restore operation.
+[discrete]
+[[monitor-restore]]
+=== Monitor a restore
+
+The restore operation uses the <<indices-recovery,recovery process>> to restore
+an index's primary shards from a snapshot. While the restore operation
+recovers primary shards, the cluster will have a `yellow`
+<<cluster-health,health status>>.
+
+After all primary shards are recovered, the replication process creates and
+distributes replicas across eligible data nodes. When replication is complete,
+the cluster health status typically becomes `green`.
+
+You can monitor the cluster health status using the <<cluster-health,cluster
+health API>>.
+
+[source,console]
+----
+GET _cluster/health
+----
+
+To get detailed information about ongoing shard recoveries, use the
+<<indices-recovery,index recovery API>>.
+
+[source,console]
+----
+GET my-index/_recovery
+----
+// TEST[setup:setup-snapshots]
+
+To view any unassigned shards, use the <<cat-shards,cat shards API>>.
+
+[source,console]
+----
+GET _cat/shards?v=true&h=index,shard,prirep,state,node,unassigned.reason&s=state
+----
+
+Unassigned shards have a `state` of `UNASSIGNED`. The `prirep` value is `p` for
+primary shards and `r` for replicas. The `unassigned.reason` describes why the
+shard remains unassigned.
+
+To get a more in-depth explanation of an unassigned shard's allocation status,
+use the <<cluster-allocation-explain,cluster allocation explanation API>>.
 
-For data streams, these index settings are applied to the restored backing
-indices.
+[source,console]
+----
+GET _cluster/allocation/explain?filter_path=index,node_allocation_decisions.node_name,node_allocation_decisions.deciders.*
+{
+  "index": "my-index",
+  "shard": 0,
+  "primary": false,
+  "current_node": "my-node"
+}
+----
+// TEST[s/^/PUT my-index\n/]
+// TEST[s/"primary": false,/"primary": false/]
+// TEST[s/"current_node": "my-node"//]
 
-// tag::index-settings-data-stream-warning[]
-[IMPORTANT]
-====
-The `index_settings` and `ignore_index_settings` parameters affect
-restored backing indices only. New backing indices created for a stream use the index
-settings specified in the stream's matching
-<<create-index-template,index template>>.
+[discrete]
+[[cancel-restore]]
+=== Cancel a restore
+
+You can delete an index or data stream to cancel its ongoing restore.
```

Comment:
`delete a __restoring__ index`

AFAICT this only makes sense for the restore & replace option. Because a deleted index, deleted again wouldn't end up with the original index. A closed index restore would delete the current data which would be unexpected... 🤔

Reply:
This isn't the case. Deleting an index or data stream also deletes its existing data, even if its a closed index. I added a statement to clarify this.